Userland/Libraries/LibWeb/HTML/Navigator.idl

```diff
@@ -6,29 +6,34 @@
 // https://html.spec.whatwg.org/multipage/system-state.html#navigator
 [Exposed=Window]
 interface Navigator {
-  // objects implementing this interface also implement the interfaces given below
+    // objects implementing this interface also implement the interfaces given below
 };
 
 // NOTE: As NavigatorContentUtils, NavigatorCookies, and NavigatorPlugins are not used in WorkerNavigator,
 //       we define them here.
 
 // https://html.spec.whatwg.org/multipage/system-state.html#navigatorcontentutils
 interface mixin NavigatorContentUtils {
-  // FIXME: [SecureContext] undefined registerProtocolHandler(DOMString scheme, USVString url);
-  // FIXME: [SecureContext] undefined unregisterProtocolHandler(DOMString scheme, USVString url);
+    // FIXME: [SecureContext] undefined registerProtocolHandler(DOMString scheme, USVString url);
+    // FIXME: [SecureContext] undefined unregisterProtocolHandler(DOMString scheme, USVString url);
 };
 
 // https://html.spec.whatwg.org/multipage/system-state.html#navigatorcookies
 interface mixin NavigatorCookies {
-  readonly attribute boolean cookieEnabled;
+    readonly attribute boolean cookieEnabled;
 };
 
 // https://html.spec.whatwg.org/multipage/system-state.html#navigatorplugins
 interface mixin NavigatorPlugins {
-  // FIXME: [SameObject] readonly attribute PluginArray plugins;
-  // FIXME: [SameObject] readonly attribute MimeTypeArray mimeTypes;
-  boolean javaEnabled();
-  readonly attribute boolean pdfViewerEnabled;
+    // FIXME: [SameObject] readonly attribute PluginArray plugins;
+    // FIXME: [SameObject] readonly attribute MimeTypeArray mimeTypes;
+    boolean javaEnabled();
+    readonly attribute boolean pdfViewerEnabled;
+};
+
+// https://w3c.github.io/webdriver/#dom-navigatorautomationinformation
+interface mixin NavigatorAutomationInformation {
```

Comment:
The comment on line 13 specifically calls out the other interfaces that are Navigator-only, should it mention this one too or be updated to be less specific?

Reply:
Fixed :+1: